data_managers/data_manager_build_bakta_database/data_manager/bakta_build_database.py

```diff
@@ -0,0 +1,255 @@
+        db_version: a string of the version number
+        in the galaxy wrapper list or just latest
+        return: info for the select or the latest bakta db version
+        """
+        try:
+            with requests.get(self.DB_VERSIONS_URL) as resp:
+                versions = json.loads(resp.content)
+        except IOError as e:
+            print(e, file=sys.stderr)
+            raise e
+        else:
+            if db_version == "latest":
+                db_date_list = []
+                for db_dic in versions:
+                    db_date_list.append(datetime.strptime(db_dic["date"],
+                                                          '%Y-%m-%d').date())
+                filtered_version = next(item for item in versions
+                                        if max(db_date_list))
+            elif db_version == "test":
+                filtered_version = {"date": "date_test",
+                                    "major": "0",
+                                    "minor": "0",
+                                    "doi": "10.5281/zenodo.7197299",
+                                    "record": "7197299",
+                                    "md5": "8b0250c17078742fc12207d4efb0fc1a",
+                                    "software-min": {"major": "0",
+                                                     "minor": "0"}
+                                    }
+            else:
+                major_version = str(db_version.split(sep=".")[0])
+                minor_version = str(db_version.split(sep=".")[1])
+                try:
+                    filtered_version = next(
+                        item for item in versions
+                        if str(item["major"]) == major_version
+                        and str(item["minor"]) == minor_version)
+                except StopIteration:
+                    print("No available version detected in the list")
+                    filtered_version = None
+            if filtered_version is not None:
+                self.db_url = f"https://zenodo.org/record/" \
+                              f"{filtered_version['record']}/files/db.tar.gz"
+                self.db_version = db_version
+                return filtered_version
+
+    def get_data_manager(self, bakta_database_info, output_path):
+        self.bakta_table_list = self.get_data_table_format()
+        bakta_value = f"V{bakta_database_info['major']}." \
+                      f"{bakta_database_info['minor']}_" \
+                      f"{bakta_database_info['date']}"
+        tool_version = str(f"{bakta_database_info['software-min']['major']}."
+                           f"{bakta_database_info['software-min']['minor']}")
+        data_info = dict(value=bakta_database_info['record'],
+                         dbkey=bakta_value,
+                         bakta_version=tool_version,
+                         path="db")
+        self.bakta_table_list["data_tables"][self.data_table_name] = data_info
```

Comment:
```suggestion
        self.bakta_table_list["data_tables"][self.data_table_name] = [data_info]
```

A DM could also add several records to a data table in a single run so this needs to be a list!

Reply:
Thank you @wm75 !
Building a DM seem to not totally clear to me, I hope improve it for the next DM.
But I don't understand how to reproduce the same result for planemo test on my local machine,
When I use :
`planemo test --biocontainers --no_dependency_resolution --no_conda_auto_init --galaxy_source https://github.com/galaxyproject/galaxy --galaxy_branch release_22.05 --galaxy_python_version 3.7 --no_cleanup`
which seem to be very close to the test used on github in CI, I can't have the same output